### PR TITLE
experiment to identify locales that are different from their respective root locales

### PR DESCRIPTION
**Pull request by caridy**
Add supports for locales that are not root locale but are different from their corresponding root locale.

### CODE REVIEW

**Comment by yahoocla** [comment]
CLA is valid!

**Comment by caridy** [comment]
related to https://github.com/yahoo/react-intl/issues/78

File: tasks/extract_cldr_data.js [hunk 1]
```diff
@@ -27,8 +27,9 @@ module.exports = function (grunt) {
 
     var dest      = this.data.dest;
     var destIsDir = path.extname(dest) === '';
-
-    var entries = options.locales.map(function (locale) {
+    var hash      = {};
+    var groups   = {};
```
Comment (okuryu) [inline tasks/extract_cldr_data.js:31]: nitpick: need one more space?

**Comment by ericf** [comment]
Closing in favor of #2

**Pull request closed without merging**